gnc/sub8_thruster_mapper/config/thruster_layout.yaml

```diff
@@ -1,66 +1,105 @@
-# This file has all of the thruster layout information needed by the sub except the
-# Effort to force mapping which is in "${find sub8_videoray_m5_thruster)/config/calibration.json"
+# This file has all of the thruster layout information needed by the sub
 # This is used when simulating or running actual thrusters.
+# CONFIDENT_MODE means that the thrusters on this port will never go offline (command packets
+#   will always be sent even if no responses are received)
 
 thruster_ports:
   - port: /dev/serial/by-id/usb-MIL_Data_Merge_Board_Ports_1_to_4_DMBP14-if00-port0
     thruster_names: [FLH, FLV]
+    CONFIDENT_MODE: False
   - port: /dev/serial/by-id/usb-MIL_Data_Merge_Board_Ports_1_to_4_DMBP14-if01-port0
     thruster_names: [FRH, FRV]
+    CONFIDENT_MODE: False
   - port: /dev/serial/by-id/usb-MIL_Data_Merge_Board_Ports_1_to_4_DMBP14-if02-port0
     thruster_names: [BLH, BLV]
+    CONFIDENT_MODE: False
   - port: /dev/serial/by-id/usb-MIL_Data_Merge_Board_Ports_1_to_4_DMBP14-if03-port0
     thruster_names: [BRH, BRV]
+    CONFIDENT_MODE: False
 
-# motor_id's should be globally unique among all thrusters
-# all vectors should be specified in body frame
+# node_id's should be globally unique among all thrusters
+# position and orientation vectors should be specified in body frame
+# calib contains the 4th to 1st order coefficients for a polynomial
+#     mapping from thrust in Newtons to effort (e = t[0]x^4 + t[1]x^3 + t[2]x^2 + t[3]x)
```

Comment:
Add a comment here explaining our temporary solution to have inaccurate high bounds / low calib numbers just to get things working. It would be confusing to read this in the future without this comment